src/main/java/de/uni_tuebingen/ub/nppm/model/EinzelbelegHatTitelKritik.java

```diff
@@ -0,0 +1,47 @@
+package de.uni_tuebingen.ub.nppm.model;
+
+import javax.persistence.*;
+import java.util.*;
+
+@Entity
+@Table(name = "einzelbeleg_hattitelkritik")
+public class EinzelbelegHatTitelKritik  {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "ID")
+    private Integer id;
+
+     @ManyToOne(targetEntity = Einzelbeleg.class)
+    @JoinColumn(name = "EinzelbelegID", referencedColumnName = "ID")
+    private Einzelbeleg einzelbeleg;
+
+
+    @ManyToOne(targetEntity = SelektionTitelKritik.class)
+    @JoinColumn(name = "TitelkritikID", referencedColumnName = "ID")
+    private SelektionTitelKritik titelKritik;
+
+    public Integer getId() {
+        return id;
+    }
+
+    public void setId(Integer id) {
```

Comment:
We will never need a setId() function because this will always be auto-generated via the database via AUTO_INCREMENT.

Reply:
kann man drin lassen, falls in zukunft benötigt, eigentlich nicht, aber schaden tut es auch nicht ... ist es ok wenn ichs drin lasse ?

Reply:
Bitte mach's lieber raus, denn wenn man es drin lässt und es "versehentlich" mal jemand benutzt wird das bei den ganzen AUTO_INCREMENT spalten die wir haben ziemlich sicher Fehler produzieren.